tmt-app/src/Components/InfoLinkPage.jsx

```diff
@@ -26,41 +29,74 @@ function MajorPage() {
       name: "BobaTalks Discord Server",
       link: "https://discord.com/invite/bobatalks",
     },
+    ...(type === "careers" && typeId
+      ? [
+          {
+            name: "Career Information via Zippia",
+            link: CAREERS_AND_MAJORS[type][typeId].link,
+          },
+        ]
+      : []),
   ];
-  let sectionInfo = [
-    {
-      title: `What is ${
-        ["a", "e", "i", "o", "u"].includes(data.name[0].toLowerCase())
-          ? "an "
-          : "a "
-      }
-      ${data.name} Major?`,
-      color: "BobaBeige.main",
-      info: data.desc,
-    },
-    {
-      title: "Classes",
-      color: "BobaPink.main",
-      info: data.classes,
-    },
-    {
-      title: `What skills do those in ${data.name} have?`,
-      color: "Lavender.main",
-      info: data.skills,
-    },
-    {
-      title: "Careers",
-      color: "BobaBeige.main",
-      info: data.careers,
-    },
-  ];
+
+  const cards = useMemo(() => {
```

Comment:
I'm not certain this needs to be memo- usually we'd look at performance. Seems like a very inexpensive calculation.

Reply:
Josh made these changes. I think he would be able to provide a better explanation.

Reply:
I'd say go ahead and test this w/o it @Raymond0T, only concern would be maintaining state with the `currentInfo` with the changes to the routing and data structures on the page, i dont think we need to maintain that variable anymore. We should be able to access the info off of the constant directly. @michachan probably right and it could be just unnecessary computation, but I dont know just by looking at this.